spec/features/heat_map_spec.rb

```diff
@@ -60,28 +60,6 @@ def create_review
     expect(page).to have_content('Summary Report for assignment')
   end
 
-  it 'should be able to follow the link to a specific review' do
```

Comment:
Why did you delete these two test cases? Or it is because they are flaky tests (sometimes pass, sometimes fail)?